azure/durable_functions/__init__.py

```diff
@@ -46,10 +47,10 @@ def validate_extension_bundles():
         # We do a best-effort attempt to detect bundles V1
         # This is the string hard-coded into the bundles V1 template in VSCode
         if version_range == "[1.*, 2.0.0)":
-            message = "Durable Functions for Python does not support Bundles V1."\
+            message = "Durable Functions for Python will stop supporting Bundles V1."\
```

Comment:
This language feels a bit strong to me, especially since we don't have a deprecation day in sight. I'd prefer something like below:

`Your application is currently configured to use Extension Bundles V1. Durable Functions for Python works best with Bundles V2, which provides additional features like Durable Entities, more bug fixes, and better performance. You can upgrade to Bundle V2 by setting extensionBundles version in your host.json to "[2.*, 3.0.0)"`

Reply:
That's fair. I'm updating the language to more-or-less your suggestion here. Thank you